docs/src/starknet/account.md

```diff
@@ -94,6 +94,23 @@ transaction: https://sepolia.starkscan.co/tx/[..]
 
 For a detailed CLI description, see [account deploy command reference](../appendix/sncast/account/deploy.md).
 
+#### Check Account Balance
```

Comment:
Don't think this fits in `Creating and Deploying Accounts`. I'd rather have a separate page on balance than mix them up like this.

Reply:
Especially since we for example have a page where we only show the usage of `sncast tx-status`.